src/client.c

```diff
@@ -68,6 +84,14 @@ int send_request(int fd, char *hostname, char *port, char *path)
   char request[max_request_size];
   int rv;
 
+  rv = sprintf(request,
```

Comment:
good figuring out that you need to pass protocol as HTTPS for it  to work with HTTPS requests 👍